packages/reactive-element/src/reactive-element.ts

```diff
@@ -837,6 +839,22 @@ export abstract class ReactiveElement
       }
     }
     this.elementStyles = this.finalizeStyles(this.styles);
+    if (DEV_MODE) {
+      if (this.hasOwnProperty('createProperty')) {
+        issueWarning(
+          'no-override-create-property',
+          'Overriding ReactiveElement.createProperty() is deprecated. ' +
+            'The override will not be called with standard decorators'
+        );
+      }
```

Comment:
I am missing something fundamental. How does this work?

E.g., I tried to create a minimal repro in a playground:

```ts

class Base {
  static createProperty() {}

  static isItOverridden() {
    if (this.hasOwnProperty('createProperty')) {
      // This incorrectly is true.
      console.log('Yes');
    }
  }
}

Base.isItOverridden(); // Logs yes???
```

Reply:
Can a test be added for this DEV_MODE warning?

Reply:
```ts
class Sub extends Base {}
Sub.isItOverridden(); // Doesn't log yes!

class Sub2 extends Base {
  static createProperty() {}
}
Sub2.isItOverridden(); // Logs yes!
```

Reply:
Peter ramped me up on what I missed as well. Thank you, it makes sense now!